gui/control_main.py

```diff
@@ -3058,6 +3086,8 @@ def measurePolyCB(self):
     def center3LoopCB(self):
         logger.info("3-click center loop")
         self.threeClickCount = 1
+        self.threeClickSignal.emit('{} more clicks'.format(str(4-self.threeClickCount)))
```

Comment:
I would suggest using f strings see section 7.1.1 - https://docs.python.org/3/tutorial/inputoutput.html . the text to emit would look like '{4-self.threeClickCount} more clicks'